styles/tooltip.less

```diff
@@ -0,0 +1,14 @@
+#hover-tooltip {
+  background-color: #226AEF;
```

Comment:
We probably want to key off the current theme's style variables. There are a few examples in our less files.

Perhaps add a TODO: here, or track this in a bug to revise this in a follow-on PR.